posts/announce-lts-22-nightly-ghc-9.8.md

```diff
@@ -0,0 +1,27 @@
+---
+author: Jens Petersen
+title: LTS 22 release for ghc-9.6 and Nightly now on ghc-9.8
+description: Stackage LTS 22 (ghc-9.6.3) has been released and Stackage Nightly moves to ghc-9.8.1.
+timestamp: 2024-01-03T06:16:00Z
+---
+
+## Stackage LTS 22 has been released
+
+The Stackage team is happy to announce that [Stackage LTS version 22](https://www.stackage.org/lts-22.0) was released last month, based on GHC stable version 9.6.3.
+
+LTS 22 includes many [package changes](https://www.stackage.org/diff/lts-21.22/lts-21.0), and has over 3300 packages!
```

Comment:
@juhp, this URL for the packages changes looks wrong to me.

Reply:
Maybe it should be this one? https://www.stackage.org/diff/lts-22.0/lts-21.0

Reply:
I think either that, or the one that compares the first of lts-22 with the last of lts-21 (lts-21.25) - the latter was the approach taken for the blog post announcing the lts-21 series.

Reply:
Thank you!  Great catch